rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/BookKeeper.java

```diff
@@ -296,6 +297,43 @@ public void onRemoval(
                 .build();
     }
 
+    @Override
```

Comment:
add a comment about this method telling that the purpose if only to warm the cache and tell how it is used